app.js

```diff
@@ -1,99 +1,114 @@
-const data = [
-  {
-    id: 1,
-    name: 'First Book',
-    author: 'First Author Name',
-  },
-  {
-    id: 2,
-    name: 'Second Book',
-    author: 'Second Author Name',
-  },
-  {
-    id: 3,
-    name: 'Third Book',
-    author: 'Third Author Name',
-  },
-  {
-    id: 4,
-    name: 'Fourth Book',
-    author: 'Fourth Author Name',
-  },
-];
-
-let localStorageData = JSON.parse(localStorage.getItem('data'));
+/* eslint-disable max-classes-per-file */
 
 const booksContainer = document.querySelector('.books-container');
-const bookForm = document.getElementById('bookForm');
+// Book class: Represent a book
+class Book {
+  constructor(bookTitle, authorName, bookId) {
+    this.bookTitle = bookTitle;
+    this.authorName = authorName;
+    this.bookId = bookId;
+  }
+}
```

Comment:
The `getBooks`, `addBooks`, and `removeBooks` method should be inside the `Book` class since they are methods of the Book class. Also try as much as possible to understand the use of `this` keyword and utilize it here.